test/expected/unsupported/types.out

```diff
@@ -3,3 +3,7 @@ ERROR:  column "a" has unsupported type
 CREATE TYPE point AS (x int, y int);
 CREATE TABLE t (a point) USING columnstore;
 ERROR:  column "a" has unsupported type
+CREATE TABLE mooncake_numeric_tbl (val numeric(40, 5)) USING columnstore;
```

Comment:
nit: can we change `mooncake_numeric_tbl` to `t` for consistency

Reply:
Updated.